DisCatSharp.ApplicationCommands/ApplicationCommandsExtension.cs

```diff
@@ -365,11 +366,19 @@ internal async Task UpdateAsync()
                             FailedGuilds.Add(guild);
                     }
                 }
-
+                //Default should be to add the help and slash commands can be added without setting any configuration
+                //so this should still add the default help
+                if (this._configuration is null || (this._configuration is not null && this._configuration.EnableDefaultHelp))
+                {
+                    foreach(var key in commands_pending.ToList())
+                    {
+                        this._updateList.Add(new KeyValuePair<ulong?, ApplicationCommandsModuleConfiguration>
+                            (key, new ApplicationCommandsModuleConfiguration(typeof(DefaultHelpModule))));
+                    }
+                }
                 if (GlobalCommands != null && GlobalCommands.Any())
                     this._globalDiscordCommands.AddRange(GlobalCommands);
-
-                foreach (var key in commands_pending)
+                foreach(var key in commands_pending.ToList())
```

Comment:
convert to list here is not necessary

Reply:
It is as I am adding to the Enumeration so without going to a list you will get the following error `Collection was modified; enumeration operation may not execute`

Reply:
ah, right. Good call I didn't think of that